package.json

```diff
@@ -95,7 +95,8 @@
     },
     "snapshotSerializers": [
       "enzyme-to-json/serializer"
-    ]
+    ],
+    "collectCoverageFrom": ["src/**/*.js"]
```

Comment:
Maybe we should add [roots](https://facebook.github.io/jest/docs/configuration.html#roots-array-string) config so it only looks for tests in the `src` directory.